### ISSUE TITLE
returns "true", "false" as string instead of boolean

### ISSUE THREAD

**Issue by felixheidecke** [nashwaan/xml-js#64]
Since `true` and `false` are expected to be boolean values, negating and/or testing explicitly failes.

XML
`<a show="false"/>`

to JSON
```
{
  "a": {
    "_attributes": {
      "show": "false"
    }
  }
}
```

Testing
```
if (!a._attributes.show) // this is will not end well
```

**Comment by winniehell** [comment]
@felixheidecke What was the reason for closing this issue?

**Comment by felixheidecke** [comment]
@winniehell … because I found out, that a `nativeType` option [already exists](https://github.com/nashwaan/xml-js#options-for-converting-xml--js-object--json).

**Comment by winniehell** [comment]
>  because I found out, that a nativeType option already exists.

However that option doesn't convert attributes. I created https://github.com/nashwaan/xml-js/pull/68 to solve that.

**Comment by nashwaan** [comment]
@winniehell thanks for your contribution. `nativeTypeAttributes` is now published in `v1.6.5`.

**Comment by winniehell** [comment]
@nashwaan Thank you very much! :heart: Especially for the fast response time.
